core/src/main/java/io/undertow/Undertow.java

```diff
@@ -250,14 +253,15 @@ public XnioWorker getWorker() {
         AJP
     }
 
-    private static class ListenerConfig {
+    public static class ListenerConfig {
```

Comment:
It's the argument to `Undertow.Builder.addListener`. Alternatively I could build a similar but slightly differnet model object, but it would be more or less identical. Any suggestions?